.github/renovate.json5

```diff
@@ -71,4 +71,10 @@
   // concurrent PRs limit.
   prConcurrentLimit: 50,
   prHourlyLimit: 5,
+
+  // schedule daily before 4 AM
+  schedule: ["* 0-3 * * *"],
```

Comment:
What some people don't like is the email noise. It's probably okay to filter emails using that label and not add this schedule. 0-3 is GMT/UTC - it's during the night for me - but not for others around the globe. Whatever schedule is in here, it will "annoy" someone.

I'd favor just the label and leave the schedule as it is.

Reply:
> I'd favor just the label and leave the schedule as it is.

+1

Reply:
I remove the schedule.